Makefile

```diff
@@ -105,8 +105,10 @@ else
 IS_CYGWIN := $(shell $(CC) -dumpmachine | grep -i cygwin | wc -l)
 ifeq ($(IS_CYGWIN),1)
 EXT = dll
-AR_EXT = lib
+AR_EXT = a
```

Comment:
why dont you keep .lib extension? 
the idea was to generate the standard library for Windows, hence .lib.